### DISCUSSION TITLE
allow adding API key in the config to Jina for search with paid account

### DISCUSSION THREAD

**Discussion by m8dhouse** [open-webui/open-webui#4109] [General]
**Is your feature request related to a problem? Please describe.**
Allow API key for Jina search to get better/faster results

**Describe the solution you'd like**
Enable API text box in the config screen and adapt code to use API key

**Comment by taylorwilsdon** [comment]
https://github.com/open-webui/open-webui/discussions/6854 would add exactly what you're looking for here but I lack the rights to submit a PR.
